### PR TITLE
Expose ubuntu-ver argument for generator.py and add extra packages for Ubuntu 18.04

### PR DESCRIPTION
**Pull request by guitarmind**
Hi there,

I saw that `ubuntu_ver` argument is supported in `to_dockerfile` function in `Composer` of generator but not exposed as command-line argument.
I made one to let users specify the desired version of Ubuntu (18.04 in my case).

Also added required packages to build on Ubuntu 18.04.

### CODE REVIEW

File: generator/modules/python.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def build(self):
             DEBIAN_FRONTEND=noninteractive $APT_INSTALL \
                 python3-pip \
                 python3-dev \
+                python3-distutils \
```
Comment (ufoym) [inline generator/modules/python.py]: Seems not working for Ubuntu 16.04.
Could you fix this issue?

Comment (guitarmind) [reply generator/modules/python.py]: @ufoym Thanks for the review! I've improved this PR for `distutils` issue in Ubuntu versions.

Tested for Ubuntu 16.04 and 18.04.
Should be good now, could you verify again? Thanks!

**Comment by guitarmind** [comment]
@ufoym Thanks for the review! I've improved this PR for `distutils` issue in Ubuntu versions.

Tested for Ubuntu 16.04 and 18.04.
Should be good now, could you verify again? Thanks!

**Comment by guitarmind** [comment]
@ufoym Thanks for merging this! Your changes are fair to me, I was just trying to keep your original module structures. Thanks.

**Comment by ufoym** [comment]
@guitarmind Thank you for your contribution:-)

**Pull request merged**